docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudwatch-firehose.asciidoc

```diff
@@ -0,0 +1,270 @@
+
+1. Visit the AWS web console and open the AWS Lambda page.
+2. Click on **Create function** and select the option to create a function from scratch.
+3. Select a **Function name**
+4. As a **Runtime**, select a recent version of Python (for example, Python 3.11).
+5. Select your **Architecture** of choice between `arm64` and `x86_64`.
+6. Confirm and create the Lambda function
+
+When AWS completes the creation of the function, visit the **Code source** section and paste the following Python code as function source code:
+
+[source,python]
+----
+import json
+
+
+def lambda_handler(event, context):
+    print("Received event: " + json.dumps(event))
+----
+
+[IMPORTANT]
+=====
+Click on **Deploy** to deploy the changes to the source code.
+=====
+
+[discrete]
+[[firehose-cloudwatch-step-two-genereate-sample-logs]]
+=== Generate some sample logs
+
+With the function ready to go, we can invoke it a few times to generate sample logs.
+
+On the function page,
+
+- Select **Test**
+- Select the option to create a new test event
+- Name the test event and **Save** the changes.
+- Click on the **Test** button to execute the function.
+
+Visit the function's log group. Usually, the AWS console offers a handy link to jump straight to the log group it created for this function's logs.
+
+You should see something similar:
+
+image::firehose-cloudwatch-sample-logs.png[CloudWatch log group with sample logs]
+
+Take note of the log group name for this Lambda function, and move to the next section.
+
+[discrete]
+[[firehose-cloudwatch-step-three]]
+== Step 3: Create a stream in Amazon Data Firehose
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to Amazon Data Firehose.  
+
+. Click *Create Firehose stream* and choose the source and destination of your Firehose stream. Unless you are streaming data from Kinesis Data Streams, set source to `Direct PUT` and destination to `Elastic`. 
+
+. Provide a meaningful *Firehose stream name* that will allow you to identify this delivery stream later. 
+
+NOTE: For advanced use cases, source records can be transformed by invoking a custom Lambda function. When using Elastic integrations, this should not be required.
+
+Set the **Parameters** in the **Destination settings** section.
+
+[discrete]
+[[firehose-cloudwatch-step-three-desination-settings-parameters]]
+=== Parameters
+
+Use the following parameters:
+
+
+[cols="1,1",options="header"]
+|===
+|Name
+|Value
+
+| `es_datastream_name`
+| `logs-aws.generic-default`
+
+|===
+
+[discrete]
+[[firehose-cloudwatch-step-four]]
+== Step 4: Set up a subscription filter to send Lambda function log events to a Firehose stream
+
+The Firehose stream is ready to send logs to our Elastic Cloud deployment.
+
+Next steps:
+
+- Open the log group with the Lambda function log events
+- Create a subscription filter for Amazon Data Firehose 
+
+[discrete]
+[[firehose-cloudwatch-step-four-log-group]]
+=== Open the log group with the Lambda function log events
```

Comment:
All of the sub-sections here make this section a bit choppy.  The task flow gets lost and there is unnecessary repetition, which IMO makes the info harder to scan and makes the procedure look more complicated. Would it make sense to have a list with sub-steps here instead of all the sub-sections? If the procedure gets too long, is it possible to break into more than one high-level, user goal oriented task?